index.md

```diff
@@ -34,13 +34,9 @@ Harassment includes, but is not limited to:
 - Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect others from intentional abuse
 - Publication of non-harassing private communication
 
-Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. We will not act on complaints regarding:
+Our open source community prioritizes marginalized people’s safety and comfort over privileged people's privileges. We will not act on complaints regarding:
 
-- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
-- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you”
-- Refusal to explain or debate social justice concepts
-- Communicating in a ‘tone’ you don’t find congenial
-- Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions
+- Any -ism from a privilege person's institutionally privileged position.
```

Comment:
Ha!

Hello 4chan, welcome to github!

https://github.com/Alcesmire/men-kampf-firefox/issues/3
https://github.com/letsgetrandy/DICSS/pull/28
https://github.com/letsgetrandy/DICSS/pull/46

Oh, I see you've already been shown around.

Well anywho, I just wanted to say hello, touch bases, What's this pr you got here, oh hey, you are trying to false flag feminism, a classic. *wholesome chuckle*

Well, anywho, wish you best of luck on your trolling.